dist/test_data.py

```diff
@@ -134,6 +134,9 @@ def __ge__(self, other):
 runtime_monitor = enabled_config_true + component_config + [
     Config('stat_cache_size', '', '''
         The maximum cache percentage that can be hit while running.''',
+        type='category', subconfig=limit_stat),
+    Config('stat_db_size', '', '''
```

Comment:
I was thinking that `stat_db_size_mb` or something would be better but now that I think about it, any user that uses this functionality would need to have a rough idea of database size anyway.